pyproject.toml

```diff
@@ -15,7 +15,7 @@ python = "^3.9"
 numpy = "^1.23.4"
 scipy = "^1.9.3"
 pyyaml = ">=6,<6.1"
-arcaflow-plugin-sdk = "^0.13.0"
+arcaflow-plugin-sdk = "~0.14.0b1"
```

Comment:
It looks like the docsgen must be using the 0.14.0b1 SDK since it was choking without the schema change, and the schema change required the new SDK. Right now there is only a beta release with the new inline discriminator feature. I'll see about us bumping that to a proper release before finalizing this.